src/server/game/AI/CreatureAI.cpp

```diff
@@ -259,7 +259,7 @@ bool CreatureAI::_EnterEvadeMode(EvadeReason /*why*/)
 
     // don't remove vehicle auras, passengers aren't supposed to drop off the vehicle
     // don't remove clone caster on evade (to be verified)
```

Comment:
Now these comments possible should be moved to [Unit.cpp](https://github.com/TrinityCore/TrinityCore/pull/16503/files#diff-4522eeffcbc961602e86620011a2d579R4121)